File: beecrowd_2344.cpp
#include <stdio.h>

int main() {
  int N;
  scanf("%d", &N);
  char indice = (N >= 86)              ? 'A'
                : (N >= 61 && N <= 85) ? 'B'
                : (N >= 36 && N <= 60) ? 'C'
                : (N >= 1 && N <= 35)  ? 'D'
                                       : 'E';

  printf("%c\n", indice);
  return 0;
}

File: BeeCrowd/Python/beecrowd_2344.py
N = int(input())
indice = 'A' if (N >= 86) else 'B' if (N >= 61 and N <= 85) else 'C' if (N >= 36 and N <= 60) else 'D' if (N >= 1 and N <= 35) else 'E'

print("%c" %indice)
